README.developers.md

```diff
@@ -668,6 +675,38 @@ k6FracN10LB_mem20K_complexDSP_customSB_22nm.xml clstm_like.large.v      common
 k6FracN10LB_mem20K_complexDSP_customSB_22nm.xml lstm.v  common  38901.96                35.76   651868  7       30532.88        -1      -1     606240   -1      -1      6626    17      305     -1      success v8.0.0-4470-ge625fdfe9  release IPO VTR_ASSERT_LEVEL=2  GNU 7.5.0 on Linux-4.15.0-124-generic x86_64    2021-06-18T14:02:07     jupiter0        /export/aman/vtr_aman/vtr-verilog-to-routing/vtr_flow/tasks     6036204 17     19       252939  204226  1       121211  7577    200     200     40000   dsp_top auto    4576.24 1453809 1136.46 3.95    8.38544 -386636 -8.385448.38544 54.87   0.944433        0.763732        237.758 176.282 -1      1876011 15      1.28987e+09     3.81683e+08     8.80433e+08     22010.877.53    284.979 214.902 -1      -1      -1      -1      -1      -1      -1      -1      -1      -1      -1      -1      -1      -1      -1     -1       -1      -1
 ```
 
+### Example: CI Tests QoR Measurement
```

Comment:
Make sure where we say how to get QoR that there is a link to this topic.  E.g.:

To generate QoR results on a suite of circuits you can either run a regression test locally (described below) or download a QoR artficact from the CI system (add link).

Reply:
Can mention that using the same machine (both CI or both not CI) is important for CPU comparisons.